app/Http/Controllers/SprintSnapshotsController.php

```diff
@@ -54,7 +54,7 @@ public function delete(SprintSnapshot $snapshot)
 
 	private function getSprintDataFactory(SprintSnapshot $snapshot)
 	{
-		$sprintData = json_decode($snapshot->data, true);
+		$sprintData = json_decode($snapshot->getData(), true);
```

Comment:
You're right that it would not be needed for routes that use model binding using `SprintSnapshot::find` to get the snapshot, however, `SprintsController` is delegating to `SprintSnapshotsController` in two places and it is needed for these cases.